README.md

```diff
@@ -1,63 +1,3280 @@
-<Name of Topic> Masterclass
-===========================
+Balena Device Debugging Masterclass
+===================================
 
+**Masterclass Type:** Core
+**Maximum Expected Time To Complete: 3 hours**
 
-[Start: Optional Depending on Prerequisites]
 # Prerequisite Classes
 
 This masterclass builds upon knowledge that has been taught in previous classes.
-To gain the most from this masterclass, we reommend that you first undertake
+To gain the most from this masterclass, we recommend that you first undertake
 the following masterclasses:
 
-* [List of hyperlinked masterclasses](https://hyperlinkedmasterclasses)
-[End: Optional Depending on Prequisites]
-
-**Masterclass Type:** <Core/Deepdive>
-**Maximum Expected Time To Complete:** <Time in minutes>
+* [balena CLI Masterclass](https://github.com/balena-io-projects/balena-cli-masterclass)
+* [balenaOS Masterclass](https://github.com/balena-io-projects/balenaos-masterclass/)
+* [balena Networking Masterclass]() *NOT YET AVAILABLE*
```

Comment:
```suggestion
* [balena Networking Masterclass](https://github.com/balena-io/networking-masterclass/) *NOT YET AVAILABLE*
```

Reply:
Even though it's not "published", might as well leave the permalink, no?

Reply:
Yep, added!